Add qt5wayland.manifest. Removed specfile change from patch.

Adapted from:

From 424c23191d81b072255bd7f178ad0ee2c2ea091a Mon Sep 17 00:00:00 2001
From: Tomasz Olszak <olszak.tomasz@gmail.com>
Date: Fri, 25 Apr 2014 14:32:22 +0200
Subject: [PATCH 10/21] Added manifest file according to smack 3-domain model.

Bug-Tizen:TIVI-2416

Change-Id: I414e245a5c8847bec32ef6a97569d0f06221e1db
Signed-off-by: Tomasz Olszak <olszak.tomasz@gmail.com>
---
 packaging/qt5-qtwayland.manifest | 5 +++++
 packaging/qt5-qtwayland.spec     | 4 ++++
 2 files changed, 9 insertions(+)
 create mode 100644 packaging/qt5-qtwayland.manifest

diff --git a/packaging/qt5-qtwayland.manifest b/packaging/qt5-qtwayland.manifest
new file mode 100644
index 0000000..75b0fa5
--- /dev/null
+++ b/packaging/qt5-qtwayland.manifest
@@ -0,0 +1,5 @@
+<manifest>
+    <request>
+        <domain name="_"/>
+    </request>
+</manifest>
-- 
1.9.1

